core/app.go

```diff
@@ -95,6 +96,8 @@ func NewApp(configFlag string) (*App, error) {
 	if err = cfg.InitLogging(); err != nil {
 		return nil, err
 	}
+	configJSON, _ := json.Marshal(*cfg)
```

Comment:
Probably not a bad idea. Note that logrus exports the log levels so you can do something like what they do in [`Logger.Debug`](https://github.com/sirupsen/logrus/blob/881bee4e20a5d11a6a88a5667c6f292072ac1963/logger.go#L178) rather than downcasing and whatnot.